topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -0,0 +1,311 @@
+>    - *"This option defines the used scoring scheme"*: `TVERSKY_REF`
+>
+{: .hands_on}
+
+This step takes about 45 minutes to an hour to complete.
+
+> ### {% icon tip %} Tip: Problems using the Align-it tool?
+> Due to the large number of files contained in the ePharmaLib dataset collection, users occasionally encounter technical issues with this particular Align-it job. Sometimes the job takes too long to successfully complete (more than the estimated 1 hour duration) depending on the available Galaxy computing resources. And in other cases the job completes with errors; if this happens, delete the errored outputs and rerun the job.
+{: .tip}
+
+# Post-processing
+
+The above pharmacophore alignment produces three types of outputs: the aligned pharmacophores (PHAR format), aligned structures (SMI format), and alignment scores (tabular format). Of these results, only the alignment scores are of interest and will be post-processed prior to analysis.
+
+## Concatenating the pharmacophore alignment scores
+
+The alignment score of the best ranked ligand conformer aligned against each ePharmaLib pharmacophore is stored in individual files. In total, this job generates 5,010 output files which should be concatenated in a single file, for a better overview of the predictions.
+
+> ### {% icon hands_on %} Hands-on 8: Concatenating the scores
+>
+> 1. {% tool [Concatenate datasets](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cat/0.1.1) %} with the following parameters:
+>    - {% icon param-file %} *"Datasets to concatenate"*: `score_result_file` (output of **Pharmacophore alignment** {% icon tool %})
+>
+{: .hands_on}
+
+## Ranking the predicted protein targets
+
+The resulting single alignment score file needs to be re-sorted according to the alignment metric, the Tversky index, i.e. the 10th column. The pharmacophores of the ePharmaLib dataset were labeled according to the following three-component code *PDBID-hetID-UniprotEntryName*. Given that staurosporine (hetID: *STU*) appears in several PDB cocrystal structures (43 times in the sc-PDB dataset from which ePharmaLib was built), we shall evaluate the predictions results by identifying the predicted proteins whereby the cocrystallized ligand is `STU`.
+
+> ### {% icon hands_on %} Hands-on 9: Sort Dataset
+>
+> 1. {% tool [Sort](sort1) %} with the following parameters:
+>    - {% icon param-file %} *"Sort Dataset"*: `out_file1` (output of **Concatenate datasets** {% icon tool %})
+>    - *"on column"*: `c10`
+>
+> 2. {% tool [Text reformatting](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_awk_tool/1.1.2) %} with the following parameters:
+>    - *"AWK Program"*: `/STU/{print NR, $0}`
+>
+>    > ### {% icon comment %} Further investigation (optional)
+>    >
+>    > The second task performed in the above *Hands-on 9* is optional and should only be performed if the query ligand possess a *PDB hetID*, i.e. is present as a cocrystallized structure in PDB structures.
```

Comment:
Can you clarify what you are doing with `awk`, and in particular what `/STU/{print NR, $0}` does?

Reply:
By modifying the input pharmacophore dataset, a different output was generated. Therefore, I deleted this section which no longer applies.